### ISSUE TITLE
How to use HTTP url for the brokenURL?

### ISSUE THREAD

**Issue by joaquintoral** [stomp-js/ng2-stompjs#110]
I used to do this back in the earlier versions

```
export function socketProvider() {
  return new SockJS('http://gmswd0101naa.delta.com:17063/gmscore/v1/gmsEvents');
}

export const stompConfig: StompConfig = {
  // Which server?
  brokerURL: socketProvider,
```

Now when I updated it for angular 7, I couldn't do it anymore.

**Comment by kum-deepak** [comment]
Are you using the compatibility mode (any of StompService/StompRService) or updated API (RxStompService)?

**Comment by gianmaria89** [comment]
i use RxStompService i have the same problem. is it possible to use RxStompService with sockjs-client?

**Comment by kum-deepak** [comment]
Yes, it is possible, please try it and report if you are facing any issues. You will need to set `webSocketFactory` in the configuration instead of `brokerURL`. Please see https://stomp-js.github.io/api-docs/latest/classes/RxStompConfig.html#webSocketFactory

**Comment by gianmaria89** [comment]
Yes, it works! thank you
